services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -430,6 +430,10 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
 
     DIRECTOR_V2_DOCKER_REGISTRY: RegistrySettings = Field(auto_default_from_env=True)
 
+    DIRECTOR_V2_SERVICES_CUSTOM_CONSTRAINTS: str = Field(
+        "", description="added to service Placement constraint in dynamic-sidecars"
```

Comment:
Could you please elaborate on what is a constraint in this context?
Preferably encode it in the type annotation (e.g. with a regex or enum, etc) but if it is too complex add at least an example with a description (with link) .. And a test?

Reply:
Let's go for an example, something like `node.labels.region==east` will be set here